OrbitBase/Platform/Linux/Profiling.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// clang-format off
+#include <OrbitBase/Platform/Linux/Profiling.h>
+// clang-format on
+
+#include <OrbitBase/Logging.h>
+
+#include <fstream>
+#include <optional>
+
+namespace {
+static std::optional<std::string> ReadFile(std::string_view filename) {
+  std::ifstream file{std::string{filename}, std::ios::in | std::ios::binary};
```

Comment:
Why `std::ios::binary` when you return a string?

Reply:
This was copied from LinuxTracingUtils and left as is. What would you suggest here?

Reply:
Even though this is reading a string we still want to read it as is - hence std::ios::binary here.
https://en.cppreference.com/w/cpp/io/c/FILE#Binary_and_text_modes

Reply:
Interesting, I didn't know that.